MANIFEST.in

```diff
@@ -36,7 +36,6 @@ include sherpa/optmethods/tests/tstopt.hh
 
 # Sherpa configuration files
 include sherpa/sherpa.rc
-include sherpa/sherpa-standalone.rc
```

Comment:
I think I'd rather see this without removing `sherpa-standalone.rc` as it's a separate issue which I think would just slow this PR down,